src/Libraries/CoreNodesUI/Input/ISlider.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel;
+
+namespace DSCoreNodesUI.Input
+{
+    public interface ISlider<T>:INotifyPropertyChanged
```

Comment:
The base classes for sliders, Double and Integer, inherit from BasicInteractive<T> already. These base classes maintain the Value property and the serialization logic for the value. For sliders, we only need to add the notion that, in addition to a Value property we need Minimum, Maximum, and Step properties as well. 

I did originally implement a base class called SliderBase, but because our double and integer classes already have their type defined, I would have to inherit instead from BasicInteractive<T>, and in so doing, would lose the logic we already have for properly serializing and deserializing int and double values. 

I'll reinvestigate to see if this can be improved.